### PR TITLE
docs: switch to RTD and improve website

### PR DESCRIPTION
**Pull request by redeboer**
Closes https://github.com/ComPWA/tensorwaves/issues/21

Output rendered at
[pwa.readthedocs.io/projects/tensorwaves/en/docs-rtd](https://pwa.readthedocs.io/projects/tensorwaves/en/docs-rtd/)

### CODE REVIEW

**Comment by redeboer** [comment]
@spflueger
Once https://github.com/ComPWA/pycompwa/pull/85 is merged, its content can be migrated here as well. Can also be done in a follow-up PR.

**Comment by redeboer** [comment]
Not that much of the instructions and info added here is similar to that of pycompwa. This will be worse once theory sections are added. A solution may be to have a 'master' page for PWA packages. But well, long term idea :)

**Comment by redeboer** [comment]
Better to do https://github.com/ComPWA/tensorwaves/pull/26 first

File: doc/installation.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Installation through PyPI
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`tensorwaves` is `available as a PyPI package <https://pypi.org/>`_ so
+installation is super easy:
+
+.. code-block:: shell
+
+  pip install tensorwaves
```
Comment (spflueger) [inline doc/installation.rst:17]: Make a comment that right now its under dev and no package exists yet

**Comment by redeboer** [comment]
API is still messed up on RTD, see e.g.
https://pwa.readthedocs.io/projects/tensorwaves/en/latest/api/tensorwaves.expertsystem.state.particle.html
What's missing is an [RTD config file](https://docs.readthedocs.io/en/stable/config-file/v2.html)... :grimacing:

File: doc/installation.rst (hunk 1 above)
Comment (redeboer) [reply doc/installation.rst:22]: https://github.com/ComPWA/tensorwaves/pull/27/commits/a619a3a80dbecff93e7e009c2070fb9016e27ad9

**Pull request merged**